Src/BootCamp.Chapter/Program.cs

```diff
@@ -1,14 +1,62 @@
 ﻿using System;
 using System.Collections.Generic;
 using System.IO;
+using System.Transactions;
 using Console = System.Console;
 
 namespace BootCamp.Chapter
 {
     class Program
+
+
     {
         static void Main(string[] args)
         {
+
+            BmiCalculator();
```

Comment:
CalculateBmi is a better name. 
Methods are verbs, not nouns.

Also, be careful with random empty lines all over the place. Just remove them. Code looks really dirty with them.